cmd/root.go

```diff
@@ -56,8 +56,7 @@ func init() {
 	// Here you will define your flags and configuration settings.
 	// Cobra supports persistent flags, which, if defined here,
 	// will be global for your application.
-	rootCmd.PersistentFlags().StringP(argRootServer, "", "", "root server URL, e.g. 'https://hosted.mender.io' (required)")
-	rootCmd.MarkPersistentFlagRequired(argRootServer)
+	rootCmd.PersistentFlags().StringP(argRootServer, "", "https://hosted.mender.io", "root server URL, e.g. 'https://hosted.mender.io' (default)")
```

Comment:
There is an acceptance test that explicitly expects a fail when not setting the server, so that should be fixed :)

Reply:
Yeh, I'll fix it up. I just wanted confirmation that this was not retarded before I bothered fixing it :)